.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -223,7 +223,7 @@ jobs:
       - name: Test
         env:
           PR_NUMBER: ${{ github.event.pull_request.number }}
-          CIRCLE_BRANCH: ${{ steps.parse-ref.outputs.branch }}
+          BRANCH: ${{ steps.parse-ref.outputs.branch }}
```

Comment:
not related to this PR but I wonder if we can just get this via the HEAD_REF variable in GHA and delete all the `parse-ref` steps, @seemethere is there any reason we didn't do that in the first place?

Reply:
I think we technically do since it normalizes the references to something that was similar to CIRCLE_BRANCH